source/modules/collection-api/sale/sale_payment.md.erb

```diff
@@ -0,0 +1,32 @@
+### Sale Payment
+```shell
+
+{
+    "method":          "credit_card",
+    "installments":    1,
+    "ccard_first_six": 455326,
+    "ccard_last_four": 0012,
+    "holder_name":     "John Fulano",
+    "holder_cpf":      "741.112.235-53",
+    "billing_address": {
+        "street_name":   "Rua Fidalga",
+        "street_number": "252",
+        "zip_code":      "05432-010",
+        "city":          "São Paulo",
+        "state":         "SP",
+        "country":       "Brasil"
+    }
+}
+```
+
+Inspetor has a standardized format for accepting payment information that is associated with a sale. You can find the accepted and required parameters below:
+
+Property             | When is it required?   | Type    | Description
+--------             | ---------------------  | ----    | -----------
+**method**           | `POST`                 | String  | The method of payment being used. *The allowed values are `credit_card`, `boleto`, and `other`*
```

Comment:
Maybe this could be a required field on `POST`?

Reply:
Hm what do you mean? isn't it already?

Reply:
It is, no idea why I said that, sorry.